db_schema/schema.md

```diff
@@ -38,3 +38,13 @@ A record that indicates that a student is assigned to some sport activity.
 - **ActivityID** ```Integer, Foreign Key``` [(SportActivity.ID)](#sport-activity)
 
 ```Primary Key``` is the tuple (**StudentID**, **ActivityID**) since it is a many-to-many relationship.
+
+### Activity Schedule
+
+- **ActivityID** ```Integer, Foreign Key``` [(SportActivity.ID)](#sport-activity)
+- **Day** ```String(3)``` (day of the week in 3-letter format (e.g. "mon"))
```

Comment:
I'd say it's better to store the weekday as an integer from 0 to 6, sorting will be easier

Reply:
Accepted